packages/yew/src/tests/layout_tests.rs

```diff
@@ -0,0 +1,142 @@
+use crate::html::{AnyScope, Scope};
+use crate::virtual_dom::{VDiff, VNode, VText};
+use crate::{Component, Context, Html};
+use gloo::console::log;
+use web_sys::Node;
+use yew::NodeRef;
+
+struct Comp;
+impl Component for Comp {
+    type Message = ();
+    type Properties = ();
+
+    fn create(_: &Context<Self>) -> Self {
+        unimplemented!()
+    }
+
+    fn update(&mut self, _ctx: &Context<Self>, _: Self::Message) -> bool {
+        unimplemented!();
+    }
+
+    fn changed(&mut self, _ctx: &Context<Self>) -> bool {
+        unimplemented!()
+    }
+
+    fn view(&self, _ctx: &Context<Self>) -> Html {
+        unimplemented!()
+    }
+}
+
+#[derive(Debug)]
+pub struct TestLayout<'a> {
+    pub name: &'a str,
+    pub node: VNode,
+    pub expected: &'a str,
+}
+
+pub fn diff_layouts(layouts: Vec<TestLayout<'_>>) {
```

Comment:
What does layout mean in this context?
From the code it does not seem like it renders anything 🤔

Reply:
nvm i looked at the tests bellow, this does seem to be comparing html